src/data/faq.json

```diff
@@ -2484,7 +2484,10 @@
                                 "anchor": "update_faq",
                                 "active": false,
                                 "textblock": [
-                                    "Updated [last-update]"
+                                    "Updated January 3rd, 2023",

```

Comment:
Is this supposed to be the date the article #update_faq was updated or is it the date that any FAQ was last updated?

If it is the first case, then I would leave it out because it could be confused with the date that any FAQ was updated.

If it is supposed to be the last date that any FAQ was updated, then that introduces the burden of having to manually update it every time anything in the FAQs is changed.

In either case it would be better to omit it.

Reply:
This is supposed to be the date that any FAQ was last updated, like intended before. Considering my other comment below, I would consider this conversation to be solved.

Reply:
This comment is resolved in https://github.com/corona-warn-app/cwa-website/pull/3302/commits/06175c5f8d1f17268641a86810c96cfd9dae6bc8. Until then it was not clear what the date was referring to.